rethinkdb/utils_common.py

```diff
@@ -178,7 +177,7 @@ def check_positive_int(opt_str, value):
 
             return int(value)
 
-        def check_existing_file(opt_str, value):
+        def check_existing_file(_, opt_str, value):
```

Comment:
Why ignoring the first parameter instead of removing the first param from the caller functions?